src/components/ProfitAndLossChart/ProfitAndLossChart.tsx

```diff
@@ -280,6 +282,19 @@ export const ProfitAndLossChart = () => {
     }
 
     return data
+      ?.map(x => {
+        if (x.operatingExpenses < 0 || x.uncategorizedOutflows < 0) {
```

Comment:
In theory there's a case where `operatingExpenses < 0` but `uncategorizedOutflows > 0` and I think we would want to handle that separately, right?

Reply:
Yes, this handles such case. For example: `operatingExpenses < 0` but `uncategorizedOutflows > 0`

![image](https://github.com/user-attachments/assets/18bfce5d-b448-4f11-bd1d-156b84979319)

When both negative: `operatingExpenses < 0` but `uncategorizedOutflows < 0` (after change suggested in the next comment)

![image](https://github.com/user-attachments/assets/f0fb519a-89e8-48c9-b6d0-1e4bd1b1a128)